x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/pages/rule_creation/index.tsx

```diff
@@ -223,8 +225,11 @@ const CreateRulePageComponent: React.FC = () => {
       );
       setDataViewOptions(dataViewIdIndexPatternMap);
     };
-    fetchDataViews();
-  }, [dataServices.dataViews]);
+    fetchDV();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => fetchDataViews(), [fetchDataViews]);
```

Comment:
Hey, what is the reason to use use callback here?

The idea here, is to just fetch data view once, when the component mount.

Why not do:

```
useEffect(() => {
    const fetchDV = async () => {...}
    fetchDV();
    };, []);
```